### ISSUE TITLE
The specified blob does not exist

### ISSUE THREAD

**Issue by wjvii** [Azure/Azure-Functions#1610]
Not sure if this is a bug or needs documentation or I just have not found the instructions.

I started a brand new Azure v3 Function project with a Timer Trigger and when I try to run it after it gets created I get Microsoft.WindowsAzure.Storage.StorageException: "The specified blob does not exist". My local.setttings.json has "AzureWebJobsStorage": "UseDevelopmentStorage=true" which was setup when I created the project. I am using Visual Studio 2019 version 16.6.0. I have Storage Emulator 5.10.

I did web searches trying to figure what what container doesn't exist but didn't have any luck and so I was not sure if it should set that up automatically in the development environment or exactly what should happen here. I have used Azure Functions a number of times, but all previous uses were with http triggers that do not require storage.

So should this setup the container itself, do I need to do it and if so are there instructions?

**Comment by wjvii** [comment]
I believe I have confirmed that this is a bug with Azure Functions 3.0 or its template in Visual Studio because if I create a new Azure Functions project but pick version 2.0 it works as expected and I see a Blob Container gets created called azure-webjobs-hosts in the Azure Storage Emulator and my timer triggered function runs. Unfortunately the 3.0 project still does not run even after 2.0 created the blob container.

Still investigating but would appreciate any input.

**Comment by wjvii** [comment]
Well now I really do not understand. I used another computer I have and both v3 and v2 Azure Functions appear to work properly. So I took the solution and project from that computer and moved them to the original system and it threw the same exception. So one one system v3 works, on the other it doesn't.

The only difference I can think of off the top of my head is that on the first system I had indicated to not use azure storage the first time through because http triggers don't require it and that is what I am used to.
